gufe/tests/test_protocol.py

```diff
@@ -725,12 +725,12 @@ def test_foreign_objects(self, units, successes):
             transformation_key=None,
         )
 
-        with pytest.raises(MissingUnitResultError, match="No such `protocol_unit`:NoDepUnit\(None\) present"):
+        with pytest.raises(MissingUnitResultError, match=r"No such `protocol_unit`:NoDepUnit\(None\) present"):
```

Comment:
Ah https://github.com/asottile/pyupgrade?tab=readme-ov-file#invalid-escape-sequences

This is nice so python doesn't think we are trying to escape things in the string